newrelic-agent/src/main/java/com/newrelic/agent/TimedTokenSet.java

```diff
@@ -37,14 +36,13 @@ public TimedTokenSet(int timeOut, TimeUnit unit, final ExpirationService expirat
         // onRemoval happens immediately after put, so we can hit onRemoval logic before getToken() even finishes
         long timeOutMilli = TimeConversion.convertToMilliWithLowerBound(timeOut, unit, 250L);
 
-        activeTokens = CacheBuilder.newBuilder()
-                .concurrencyLevel(8)
+        activeTokens = Caffeine.newBuilder()
+                .initialCapacity(8)
```

Comment:
Here and further down: Caffeine [doesn't expose a `concurrencyLevel` setting](https://github.com/ben-manes/caffeine/blob/7f98672657a6e3515eac00ec86990b877a9d10dc/guava/src/test/java/com/google/common/cache/CacheBuilderFactory.java#L144-L150).  While working on this PR I found one source recommending to set the `initialCapacity` instead, but now I can't find it back.

CC @ben-manes in case this change makes no sense.

Reply:
The FAQ entry on [Write Contention](https://github.com/ben-manes/caffeine/wiki/Faq#write-contention) does recommend increasing the `initialCapacity` if you want to speed up a warm-up phase. The number of locks is based on the hash table capacity (the hashbin is used as a lock), so resizes increases the lock count automatically. When the cache and many threads try to write to it, they will more likely collide on the same set of locks. The default is 16.

Reply:
Ah, that's where I read it indeed: 
> consider increasing the initial capacity to your expected maximum concurrency to compensate

Thanks as always :) I'll let the New Relic devs provide feedback on this, as I'm not familiar with this code's behavior at runtime.